kobo/apps/subsequences/integrations/google/google_transcribe.py

```diff
@@ -3,63 +3,182 @@
 import posixpath
 from concurrent.futures import TimeoutError
 from datetime import timedelta
+from typing import Union, Tuple, Any
```

Comment:
`Tuple` is deprecated. Use `from __future__ import annotations` and use lowercase type everywhere instead.
See https://docs.python.org/3/library/typing.html#deprecated-aliases